testsuite/features/secondary/minkvm_guests.feature

```diff
@@ -230,7 +230,7 @@ Feature: Be able to manage KVM virtual machines via the GUI
     Given I am on the "Virtualization" page of this "kvm_server"
     When I follow "Storage"
     And I follow "Create Pool"
-    And I wait until I see "General" text
+    And I wait until option "dir" is available as "type"
```

Comment:
"appears in list" less vague than "is available as" ?

it would stress it is in ui

Reply:
done